test/dispatch_web/home/controller_test.exs

```diff
@@ -0,0 +1,18 @@
+defmodule DispatchWeb.Home.ControllerTest do
+  use DispatchWeb.ConnCase
+
+  import Mox
+
+  setup :verify_on_exit!
+
+  test "GET /", %{conn: conn} do
+    expect(Dispatch.Settings.MockClient, :stacks, fn -> ["foo", "bar"] end)
```

Comment:
_This question is unrelated to the PR itself!_

Why are we defining mocks in a compiled file (_ie_ `test/support/mocks.ex`) instead of the `test_helper.exs` script?